lib/components/Input/text-field/text-field.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import { Meta, StoryObj } from "@storybook/react";
+import TextField from ".";
+import {
+    StandaloneCircleCheckBoldIcon,
+    StandaloneTriangleExclamationBoldIcon,
+} from "@deriv/quill-icons";
+
+const meta = {
+    title: "Components/Inputs/Text Field",
+    component: TextField,
+    parameters: {
+        layout: "centered",
+    },
+    tags: ["autodocs"],
+    args: {
+        inputSize: "md",
+        status: "neutral",
+        disabled: false,
+        variant: "fill",
+        textAlignment: "left",
```

Comment:
`inputSize`, `status` and `textAlignment` can be also removed from here because these are default values of the Input component, right? :) Will it work if you remove them?

Reply:
yes, it will work if we remove. the control won't be available in the storybook